cheatsheets_draft/OAuth_Cheat_Sheet.md

```diff
@@ -1,10 +1,10 @@
-# OAuth Cheat Sheet
+# Introduction
 
-This CS will handle concepts related to using OAuth in different ways and methods.
+Into blurb.
 
-## Table of Contents
+# Contents
```

Comment:
`## Contents`, following the linter. It is recommended to have one h1 header. Other CSs are slowly being updated to follow this practice.

Reply:
Addressed.